### DISCUSSION TITLE
Remove [Not 24/7] status

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#522] [Ideas]
At this moment more than 3000 links are marked as "Not 24/7" and since quickly determine whether the broadcast is temporarily down or the link is just dead is not possible they can stay in the playlist indefinitely.

In this regard, there is a proposal to completely abandon the label Not 24/7 and instead add to the title the exact time of the broadcast. Something like this:

```
#EXTINF:-1,Example TV (Fr. 10:00-23:00)
http://example.com/playlist.m3u8
```

This way users will know when to expect the broadcast and we will be able to filter out bad links.

1. 👍 Good idea
2. 👎 Bad idea
3. 😕 Don't know
 
P.S. If you know a simpler solution to this problem, please share it in the comments

**Comment by sguinetti** [comment]
In theory it seems like a good idea, but there are channels such as La1 or Latina that decide what programs or events to broadcast and what programs no. For example, the international stream does not broadcast foreign productions in an unpredictable time. Therefore, I do not find schedules effective to some channels.

Another case is temporary maintenance as Streamlink services (for Youtube, Dailymotion and other) and holiday breaks such as RT without losing its url stream. (edited)

I propose like "(Scanned: 8/3) [Not 24/7]" (month/day) to mark the last time the bot checked the stream before [Not 24/7] status. The date is updated daily, when the bot checks that it is online.

This way, those that have not been transmitted for a long time are manually checked before being eliminated because there are easier to find by their date of last checked connection. Or if the streams did not register a online status at least once a month for update their date of last checked connection and there was no manual intervention, they will be marked to [Offline] automatically.

In case there is no "scanned" info in Not 24/7 stream, it will be automatically generated on the date it was included in the list (like today).

**Reply by freearhey** [reply]
But in these cases, if I understand correctly, the broadcasts themselves do not stop working, they show at least the splash screen. It is the same with channels that broadcast only events. In these cases, you do not need to add any label.

I'm talking about those cases where the broadcast is available only at certain times of the day or week, and at other times the server returns error 404 Not Found.

As for keeping statistics of checks, I am also considering this option.

**Reply by sguinetti** [reply]
I agree that the software should be in charge of checking the last conection of 24/7 channels. This will have a log to identify channels by last check date.
